<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>115</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>325</width>
    <height>115</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>325</width>
    <height>115</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Авторизация</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../res/if_users_white.png</normaloff>../res/if_users_white.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-color:#3a3b3a;
	color:#ffffff;
	border-color:#ffffff;
}
QTableView{
	background-color:#ffffff;
	alternate-background-color:#e2e2e2;
	color:#3a3b3a;
	border-style:solid;
	border-color:#ffffff;
	border-width:0px;
	selection-color:#ffffff;
	selection-background-color:#54b0cb;
}
QHeaderView::section{
    background-color:#448fa6;
	color:#ffffff;
	font:bold;
	height:40px;
	border: #ffffff;
	border-style:solid;
	border-width:0px;
}
QComboBox{
	background-color:#ffffff;
	border-style:solid;
	border-width:2px;
	border-color:#ffffff;
	color:#3a3b3a;
	width:50px;
	height:30px;
	font:bold;
	padding-left:5px
}
QComboBox QAbstractItemView {
	background-color:#ffffff;
	color:#3a3b3a;
    border: 2px solid #ffffff;
	padding-left:2px;
	padding-right:2px;
	padding-top:2px;
	padding-bottom:2px
}
QComboBox::drop-down {
   width: 24px;
   border: 2px;
}
QComboBox::down-arrow {
    image: url(&quot;./res/if_chevron-left.png&quot;);
    width: 16px;
    height: 16px;
}
QComboBox::down-arrow::hover {
    image: url(&quot;./res/if_chevron-down.png&quot;);
    width: 16px;
    height: 16px;
}
QDateEdit{
	background-color:#ffffff;
	border-style:solid;
	border-width:2px;
	border-color:#ffffff;
	color:#3a3b3a;
	width:150px;
	height:30px;
	font:bold;
	padding-left:5px
}
QDateEdit QAbstractItemView {
	background-color:#ffffff;
	color:#3a3b3a;;
}
QDateEdit::drop-down {
   width: 24px;
   border: 2px;
}
QDateEdit::down-arrow {
    image: url(&quot;./res/if_chevron-left.png&quot;);
    width: 16px;
    height: 16px;
}
QDateEdit::down-arrow::hover {
    image: url(&quot;./res/if_chevron-down.png&quot;);
    width: 16px;
    height: 16px;
}
QMenuBar{
	background-color:#3a3b3a;
	color:#ffffff;
	selection-color:#54b0cb;
	font:bold
}
QMenuBar::item::selected{
	background-color:#54b0cb;
	color:#ffffff;
	font:bold;
	padding-left:2px
}
QToolBar{
	background-color:#3a3b3a;
	color:#ffffff;
}
QMenu {
	background-color:#3a3b3a;
	color:#ffffff;
	font:bold;
	padding-left:2px;
}
QMenu::hover {
    background-color: #448fa6;
    color:#3a3b3a;
	font:bold
}
QMenu::item {
    background-color:#3a3b3a;
	color:#ffffff;
}
QMenu::item::selected{
	background-color:#54b0cb;
	color:#ffffff;
	padding-left:10px;
}
QTabWidget{
	background-color:#3a3b3a;
	color:#ffffff;
	border-color:#ffffff;
}
QTabBar::tab:right:selected{
    background-color:#448fa6;
	color:#ffffff;
	font:bold;
	height:60px;
	width: 30px;
	border: #448fa6;
	border-style:solid;
	border-width:2px;
}
QTabBar::tab:right:!selected{
    background-color:#3a3b3a;
	color:#ffffff;
	font:bold;
	height:60px;
	width: 30px;
	border: #448fa6;
	border-style:solid;
	border-width:2px;
}
QLabel{
	background-color:#448fa6;
	color:#ffffff;
	height:60px;
	width: 30px;
	border: #ffffff;
	border-style:solid;
	border-width:5px;
}
QGroupBox{
	background-color:#448fa6;
	color:#ffffff;
	height:60px;
	width: 30px;
	border: #ffffff;
	border-style:solid;
	border-width:5px;
}
QPushButton{
	background-color:#3a3b3a;
	color:#ffffff;
	height:60px;
	width: 40px;
	border: #ffffff;
	border-style:solid;
	border-width:5px;
	font:14px;
	font-weight: bold
}
QPushButton::pressed{
	background-color:#ffffff;
	color:#3a3b3a;
	border:2px solid #3a3b3a;
}
QPushButton::hover{
	color:#448fa6;
	border:2px solid #448fa6;
}
QStatusBar{
	background-color:#3a3b3a;
	color:#ffffff;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="loginEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>231</width>
      <height>32</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhLowercaseOnly</set>
    </property>
    <property name="placeholderText">
     <string>Имя пользователя</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="passEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>231</width>
      <height>32</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Пароль</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="userLabel">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>4</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../res/if_users_white.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="errorLabel">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>4</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../res/if_alert-triangle_white_64.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="loginAction">
   <property name="icon">
    <iconset>
     <normaloff>../res/if_log-in.png</normaloff>../res/if_log-in.png</iconset>
   </property>
   <property name="text">
    <string>Вход в систему</string>
   </property>
   <property name="toolTip">
    <string>Войти</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>loginEdit</tabstop>
  <tabstop>passEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
